BUG new unit test failed

While the Py3.10 CI passed, the Py3.8 CI failed at this point:

```py
    def test_sim_heater(method, scan, Kp, Ki):
        epid = Fb_EpidDatabaseHeaterSimulator(f"{IOC}epid1", name="epid")
        epid.wait_for_connection()
    
        getattr(epid, method)()
        assert epid.scanning_rate.get() == scan
        assert epid.proportional_gain.get() == Kp
>       assert epid.integral_gain.get() == Ki
E       AssertionError: assert 0.0 == 0.5
```

Other similar failures were encountered in development (and the test asserts removed).

Next time, failed Py3.11 at different code:

```py
_______________________________ test_sseq_reset ________________________________

    def test_sseq_reset():
        user = UserStringSequenceDevice(IOC, name="user")
        user.wait_for_connection()
        user.enable.put("Enable")
        assert len(user.read()) == 200
    
        sseq = user.sseq10
        assert isinstance(sseq, SseqRecord)
        sseq.enable.put("E")  # Note: only "E"
    
        step = sseq.steps.step1
        assert isinstance(step, sseqRecordStep)
    
        step.reset()
        timed_pause()
    
        assert step.input_pv.get() == ""
    
        uptime = EpicsSignal(f"{IOC}UPTIME", name="uptime")
        uptime.wait_for_connection()
    
        step.input_pv.put(uptime.pvname)
        assert step.string_value.get() == f"{0:.5f}"
        sseq.process_record.put(1)
        assert step.string_value.get() <= uptime.get()
    
        user.reset()
        timed_pause()
>       assert step.input_pv.get() == ""
E       AssertionError: assert 'gp:UPTIME NPP NMS' == ''
E         + gp:UPTIME NPP NMS
```

Re-run the tests.

--- a/apstools/synApps/tests/test_epid.py
+++ b/apstools/synApps/tests/test_epid.py
@@ -1,13 +1,15 @@
 import pytest
 
 from ...tests import IOC
+from ...tests import timed_pause
 from ..epid import EpidRecord
 from ..epid import Fb_EpidDatabase
 from ..epid import Fb_EpidDatabaseHeaterSimulator
 
 
 @pytest.mark.parametrize(
-    "support", [EpidRecord, Fb_EpidDatabase, Fb_EpidDatabaseHeaterSimulator],
+    "support",
+    [EpidRecord, Fb_EpidDatabase, Fb_EpidDatabaseHeaterSimulator],
 )
 def test_connection(support):
     """Connection test."""
@@ -28,6 +30,7 @@ def test_sim_heater(method, scan, Kp, Ki):
     epid.wait_for_connection()
 
     getattr(epid, method)()
+    timed_pause()
     assert epid.scanning_rate.get() == scan
     assert epid.proportional_gain.get() == Kp
     assert epid.integral_gain.get() == Ki

